spec/fixtures/aspace/generated/publicpolicy/MC148.processed.EAD.xml

```diff
@@ -53,55 +62,51 @@
bjects/346691</unitid><physdesc altrender="whole"><extent altrender="materialtype spaceoccupied">4 boxes</extent></physdesc><unitdate datechar="creation" normal="1947-01-01/1952-12-31" type="inclusive">1947-1952</unitdate></did></c></c><c id="aspace_MC148_c02577" level="series"><did><unittitle>Series 18: Correspondence and Related Materials</unittitle><unitid type="aspace_uri">/repositories/3/archival_objects/346670</unitid><physdesc altrender="whole"><extent altrender="materialtype spaceoccupied">208 boxes</extent></physdesc><physdesc altrender="whole"><extent altrender="materialtype spaceoccupied">2 items</extent></physdesc><unitdate datechar="creation" normal="1905-10-19/1981-12-31" type="inclusive">1905 October 19-1981</unitdate></did><scopecontent id="aspace_cf1d6e73e00d875751e9104248832c97"><head>Scope and Contents</head><p>Series 18: Correspondence and Related Materials consists of mostly professional correspondence to and from Lilienthal from 1905 to 1981. Some correspondence relates to particular subjects and are labeled as such (e.g., Argentina) rather than as correspondence to or from a particular individual.</p></scopecontent><arrangement id="aspace_24ef79383058c25d28002ffeb59ee3a7"><head>Arrangement</head><p>The materials in this series are arranged in chronological order by the last name of the correspondent or subject.</p></arrangement><c id="aspace_MC148_c07360" level="subseries"><did><unittitle>Subseries 18N: 1977-1978</unittitle><unitid type="aspace_uri">/repositories/3/archival_objects/346830</unitid><physdesc altrender="whole"><extent altrender="materialtype spaceoccupied">15 boxes</extent></physdesc><unitdate datechar="creation" normal="1977-01-01/1978-12-31" type="inclusive">1977-1978</unitdate></did><scopecontent id="aspace_6bd687d8d60038e5ff9ff224eef42e9c"><head>Scope and Contents</head><p>This subseries contains correspondence and related material for 1977 and 1978.</p></scopecontent><arrangement id="aspace_0632218382667cd5ff5210f4dcebc941"><head>Arrangement</head><p>The materials in this subseries are arranged in chronological order by year and within each year alphabetically by last name of correspondent or subject.</p></arrangement><c id="aspace_MC148_c07608" level="file"><did><unittitle>Center of Theological Inquiry</unittitle><unitid type="aspace_uri">/repositories/3/archival_objects/353879</unitid><physdesc altrender="whole"><extent altrender="materialtype spaceoccupied">1 box</extent></physdesc><unitdate datechar="creation" normal="1978/1978" type="inclusive">1978</unitdate><container altrender="scamudd" encodinganalog="NBox" id="aspace_62cdc0b74d1cedf000de39fc7e5b47ae" label="Mixed Materials [32101105993479]" type="box">537</container><dao xlink:actuate="onRequest" xlink:href="https://webspace.princeton.edu/users/mudd/Digitization/MC148/MC148_c07608.pdf" xlink:role="" xlink:show="new" xlink:title="View digital content" xlink:type="simple"><daodesc><p>View digital content</p></daodesc></dao></did></c></c></c></dsc>
```

Comment:
For reference. This looks like the section that adds the link that was observed to be broken.
```
xlink:href="https://webspace.princeton.edu/users/mudd/Digitization/MC148/MC148_c07608.pdf" xlink:role=""
```

Reply:
And the spec that fails without the code change is spec that fails without this change is spec/features/catalog_view_spec.rb:230